core/arch/arm/mm/tee_pager.c

```diff
@@ -532,9 +532,14 @@ void tee_pager_add_core_area(vaddr_t base, enum tee_pager_area_type type,
 	uint32_t flags = 0;
 	size_t fobj_pgoffs = 0;
 	vaddr_t b = base;
-	size_t s = fobj->num_pages * SMALL_PAGE_SIZE;
+	size_t s;
```

Comment:
Please assign a default value to `s`.

Reply:
The default value is never used

Reply:
you are right, but it is optee convention that expects all variables to get a default where defined. Compiler will take care of discarding useless default values.
See https://optee.readthedocs.io/en/latest/general/coding_standards.html, item 3.

Reply:
fixed